src/common/SettingsAPI/settings_helpers.cpp

```diff
@@ -5,6 +5,7 @@ namespace PTSettingsHelper
 {
     constexpr inline const wchar_t* settings_filename = L"\\settings.json";
     constexpr inline const wchar_t* oobe_filename = L"oobe_settings.json";
+    constexpr inline const wchar_t* last_version_run_filename = L"oobe_last_version_run.json";
```

Comment:
Why didn't you place a new setting in the `oobe_settings.json`?

Reply:
The two settings end up being used for different things, I think.
They end up being written in different places and care only for their one off values, not really filling a structure.
The idea is keeping code simpler here, while following what seems to be the logic for these one off value struct files in the root, including `log_settings.json`, `oobe_settings.json`, `settings-telemetry.json` and eve `settings-placement.json`.

Reply:
Changed the name of the file in the latest commit. Thank you for the review!